example/MODULE.bazel

```diff
@@ -182,23 +172,39 @@ use_repo(ruby, "bundle", "ruby", "ruby_toolchains")
 
 register_toolchains("@ruby_toolchains//:all")
 
-# Download cppcheck premium tar files for different platforms
-http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-# Even though cppcheckpremium is downloaded, it behaves like the free version, if we do not
-# provide a cppcheck.cfg file (see cppcheck.BUILD file) and license key.
-http_archive(
-    name = "cppcheck_linux",
-    build_file = "//tools/lint:cppcheck.BUILD",
-    integrity = "sha256-IqQ3Iofw6LoHh4YcdbN0m3tjg6utCiey7nGaOaPMv/I=",
-    strip_prefix = "cppcheckpremium-25.8.3",
-    urls = ["https://files.cppchecksolutions.com/25.8.3/ubuntu-22.04/cppcheckpremium-25.8.3-amd64.tar.gz"],
+format_tools = use_extension("@aspect_rules_lint//format:extensions.bzl", "tools")
+format_tools.google_java_format()
+format_tools.ktfmt()
+format_tools.swiftformat()
+format_tools.taplo()
+use_repo(format_tools, "google-java-format", "ktfmt", "swiftformat", "swiftformat_mac", "taplo")
+
+lint_tools = use_extension("@aspect_rules_lint//lint:extensions.bzl", "tools")
+lint_tools.checkstyle()
+lint_tools.cppcheck()
+lint_tools.spotbugs()
+lint_tools.ktlint()
+lint_tools.pmd()
+lint_tools.vale()
+lint_tools.vale_styles(
+    styles = [
+        "Google",
+        "write-good",
+    ],
 )
-
-http_archive(
-    name = "cppcheck_macos",
-    build_file = "//tools/lint:cppcheck.BUILD",
-    integrity = "sha256-PEtm/DxKNZNJJuZE+56AZ80R22sZjZoziekAmR7FhNk=",
-    strip_prefix = "cppcheckpremium",
-    urls = ["https://files.cppchecksolutions.com/25.8.3/cppcheckpremium-25.8.3-macos-15.tar.gz"],
+use_repo(
+    lint_tools,
+    "com_github_pinterest_ktlint",
+    "com_puppycrawl_tools_checkstyle",
+    "cppcheck_linux",
+    "cppcheck_macos",
+    "net_sourceforge_pmd",
+    "spotbugs",
+    "vale_Google",
+    "vale_Linux_64-bit",
+    "vale_Linux_arm64",
+    "vale_Windows_64-bit",
```

Comment:
The mixed case binaries 😨

Reply:
yeah well... that's existing and I don't think it's bad enough to break any users